server/src/cm_cmd_exec.cpp

```diff
@@ -159,7 +158,7 @@ void find_and_parse_cub_admin_version (int &major_version, int &minor_version)
   char cmd_name[CUBRID_CMD_NAME_LEN];
 
   cubrid_cmd_name (cmd_name);
-  snprintf (tmpfile, PATH_MAX - 1, "%s/cub_admin_version", sco.dbmt_tmp_dir);
+  make_temp_filepath (tmpfile, sco.dbmt_tmp_dir, "cub_admin_version", 0, PATH_MAX);
```

Comment:
task_code is a number, not an enum type.
It needs to be changed to enum type.

Reply:
Thank you for your comment!
It is changed as you recommended.